doc/man3/OSSL_PARAM_int.pod

```diff
@@ -232,15 +231,16 @@ will be assigned the size the parameter's I<data> buffer should have.
 
 OSSL_PARAM_get_utf8_string() retrieves a UTF8 string from the parameter
 pointed to by I<p>.
-The string is either stored into I<*val> with a length limit of I<max_len> or,
+The string is either stored into I<*val> with a size limit of I<max_len> or,
```

Comment:
Could you please add something like `Terminating NUL is included in the size limit.`? I feel it might be unclear to readers otherwise.

Reply:
Thanks for the request, that had me look at the function, and to find a bug.  I fixed that along with this piece of documentation.